services/scan/src/importer.test.ts

```diff
@@ -162,6 +163,7 @@ test('restoreConfig reconfigures the interpreter worker', async () => {
     workers.Input,
     workers.Output
   >(workerCall);
+  workspace.store.setElection(electionSampleDefinition);
```

Comment:
In a a few places, I have to set the election in tests because now we have the constraint that configs depend on an election existing.